Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/detail/AgendaDetailFragment.kt

```diff
@@ -108,7 +109,10 @@ class AgendaDetailFragment : Fragment() {
 
 
     private fun populateSpeakersInformation(itemData: ScheduleRow) = when {
-        itemData.speakerNames.size == 0 -> tv_agenda_detail_speaker_name.visibility = View.GONE
+        itemData.speakerNames.size == 0 -> {
```

Comment:
like `isEmpty` more, but not required

Reply:
"description" is probably implied... feedback from @nisrulz  and @ragdroid ?

Good point about the isEmpty()...Android Studio usually reminds me when the same thing is done in Java...guess the Kotlin hints aren't as good yet :-)

Noticed while using the detail screen that the details seem to take a while to load up too...still have to look at why...